transaction-spec/src/test/java/com/bloxbean/cardano/client/transaction/util/TransactionUtilTest.java

```diff
@@ -67,4 +67,13 @@ public void getTxHash_fromBytes() {
+        var txBodyHex = HexUtil.encodeHexString(TransactionUtil.extractTransactionBodyFromTx(HexUtil.decodeHexString(cbor)));
+        System.out.println(txBodyHex);
```

Comment:
Just less than a minor thing: I think the System.out will just output a hex into the console which doesn't help the user at all. So it's not needed from my point of view, just to keep it clean.

Reply:
Thanks. I will remove it.